cmd/admin-handlers.go

```diff
@@ -2896,6 +2896,36 @@ func (a adminAPIHandlers) InspectDataHandler(w http.ResponseWriter, r *http.Requ
 	}
 	sb.WriteString("\n")
 	logger.LogIf(ctx, embedFileInZip(inspectZipW, "inspect-input.txt", sb.Bytes()))
+
+	// save MinIO start script to inspect command
+	var scrb bytes.Buffer
+	scrb.WriteString(`#!/usr/bin/env bash
+
+function main() {
+	for file in $(ls -1); do
+		dest_file=$(echo "$file" | cut -d ":" -f1)
+		mv "$file" "$dest_file"
+	done
+
+	# Read content of inspect-input.txt
+	MINIO_OPTS=$(grep "Server command line args" <./inspect-input.txt | sed "s/Server command line args: //g" | sed -r "s#https:\/\/#\.\/#g")
```

Comment:
@harshavardhana I understand we should not simply take all the paths here and start server. In extracted dirs of the inspect file, if certain dir dont have a xl.meta file, we should set the permissions of those dirs as 644 so that when server comes up, it doesn't write the missing parts and we can achieve the actual stage where the source server state was.
Kindly comment or guide on this aspect so that this script can incorporate the same.

Reply:
I am not sure why we are generating a script inside the code itself ? can't we just write a separate script under github.com/minio/minio and pass inspect.enc as argument ?

Reply:
There was a suggestion to include as part of inspect-data.enc, so post extraction user can just run `./start-minio.sh`